.gitlab-ci.yml

```diff
@@ -72,24 +72,36 @@ check:nix-dry:
     - if: $CI_COMMIT_BRANCH && $CI_COMMIT_BRANCH != 'master' && $CI_COMMIT_TITLE !~ /^[0-9]+\.[0-9]+\.[0-9]+(?:-.*[0-9]+)?$/
       when: manual
 
-check:test:
+check:test-generate:
   stage: check
-  needs: []
```

Comment:
You should still leave this in, mainly cause we are always using DAG for all jobs, and `needs: []` indicates it is a "leaf" job.